index.js

```diff
@@ -399,10 +456,12 @@ ko.bindingHandlers.fastForEach = {
         $context: context
       });
     }
+    if (FastForEach.DEBUG)
```

Comment:
I'm about to eliminate this DEBUG stuff. However, I don't have any idea how I could expose the FFE instance via the element for testing. Can you see any general and nice way for it without DEBUG checking and performance penalty?